.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @sous-chefs/maintainers
```

Comment:
This is showing an error; maybe the team isn't visible or added to the repo or something?

Reply:
Yeah, probably a temporary error. It's a copied in file and works everywhere else so I'm going to say ignore it for now 👍🏼